lib/runner/test-runners/default.js

```diff
@@ -45,6 +45,12 @@ class DefaultRunner {
    * @return {Promise}
    */
   closeOpenSessions(err) {
+    if (this.playgroundServer) {
+      Logger.info(`Attempting to close websocket server running on port ${this.playgroundServer.portNumber}...`);
+      
+      this.playgroundServer.closeSocket();
+    }
```

Comment:
keep this inside a function in PlayGroundServer

Reply:
Are you asking to move the Logger? Because if we move everything how will I receive the trigger to close the socket?

Reply:
But `closeSocket` function does the same job which is written in `lib/testsuite/selector-playground/playground-listener.js` file